connectors/src/connectors/intercom/index.ts

```diff
@@ -740,3 +740,19 @@ export async function retrieveIntercomContentNodeParents(
 
   return new Ok(parents);
 }
+
```

Comment:
This looks good and should pause the cron sync of the Help Center. 
Should we also pause the webhook processing for conversations here? https://github.com/dust-tt/dust/blob/main/connectors/src/api/webhooks/webhook_intercom.ts#L91-L123

Reply:
You've done that already!! 🤦🏻‍♀️  Closing this 😬